src/main/java/gift/dto/request/KakaoLink.java

```diff
@@ -0,0 +1,4 @@
+package gift.dto.request;
+
+public record KakaoLink(String web_url, String mobile_web_url) {
```

Comment:
자바 코드 컨벤션의 네이밍 전략을 지키면서도 API의 스펙에도 맞출 수 있도록  `JsonProperty`를 활용해볼 수 있을 것 같아요. 
(이미 이전에도 활용해보신 적이 있으니 금방 이해가 되실 것 같네요. 😃)

```suggestion
public record KakaoLink(
    @JsonProperty("web_url")
    String webUrl, 
    
    @JsonProperty("mobile_web_url")
    String mobileWebUrl
) {
```